src/logiDice.js

```diff
@@ -73,13 +73,16 @@ module.exports = {
 			return Promise.all(recPromises).then((subQueries) => {
 				let diceResult = subQueries.reduce((tally, current, index) => {
 					return mergeResults(tally, current);
-				}, false);
+				}, {
+					rolls: [],
```

Comment:
This makes sense, but why not expand the sanity check on line 58? Maybe that needs to be removed now, since it'll behave differently if it wasn't for the 'd' and 'x' in 'index'?